docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Configuration
+sidebar_order: 6100
+description: "Learn about the User Feedback Widget configuration options."
+---
+
+## User Feedback Widget
```

Comment:
Since this page is only about `Feedback Widget` we can remove this heading and bump all the other headings one up to lower the depth.

Reply:
Good idea. Updated with f450cba5279d84ea908f5a8946579431aa1540ee